test_dir/new_file_1

```diff
@@ -0,0 +1 @@
+lets see, here is the file
```

Comment:
RiflerRick committed on a restricted repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) authored by @RiflerRick in the file test_dir/new_file_1
Owners: @